src/System.Net.Mail/ref/System.Net.Mail.cs

```diff
@@ -9,12 +9,12 @@ namespace System.Net.Mail
 {
     public partial class AlternateView : System.Net.Mail.AttachmentBase
     {
-        public AlternateView(System.IO.Stream contentStream) : base (default(string)) { }
-        public AlternateView(System.IO.Stream contentStream, System.Net.Mime.ContentType contentType) : base (default(string)) { }
-        public AlternateView(System.IO.Stream contentStream, string mediaType) : base (default(string)) { }
-        public AlternateView(string fileName) : base (default(string)) { }
-        public AlternateView(string fileName, System.Net.Mime.ContentType contentType) : base (default(string)) { }
-        public AlternateView(string fileName, string mediaType) : base (default(string)) { }
+        public AlternateView(System.IO.Stream contentStream) : base (default(System.IO.Stream)) { }
+        public AlternateView(System.IO.Stream contentStream, System.Net.Mime.ContentType contentType) : base (default(System.IO.Stream)) { }
+        public AlternateView(System.IO.Stream contentStream, string mediaType) : base (default(System.IO.Stream)) { }
+        public AlternateView(string fileName) : base (default(System.IO.Stream)) { }
+        public AlternateView(string fileName, System.Net.Mime.ContentType contentType) : base (default(System.IO.Stream)) { }
+        public AlternateView(string fileName, string mediaType) : base (default(System.IO.Stream)) { }
```

Comment:
Why is the type changing from 'string' to 'System.IO.Stream' here?  This now doesn't match the actual implementation.